File: java/src/test/java/com/alxarafe/domain/inventory/valueobject/HuStatusTest.java
package com.alxarafe.domain.inventory.valueobject;

import com.alxarafe.app.domain.inventory.valueobject.HuStatus;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class HuStatusTest {

    @Test
    void values() {
        assertEquals(HuStatus.AVAILABLE, HuStatus.valueOf("AVAILABLE"));
        assertEquals(HuStatus.IN_TRANSIT, HuStatus.valueOf("IN_TRANSIT"));
        assertEquals(HuStatus.BLOCKED, HuStatus.valueOf("BLOCKED"));
        assertEquals(3, HuStatus.values().length);
    }
}

File: php/tests/Unit/Domain/Inventory/ValueObject/HuStatusTest.php
<?php
declare(strict_types=1);

namespace Tests\Unit\Domain\Inventory\ValueObject;

use Alxarafe\App\Domain\Inventory\ValueObject\HuStatus;
use PHPUnit\Framework\TestCase;

final class HuStatusTest extends TestCase
{
    public function testValues(): void
    {
        self::assertSame('AVAILABLE', HuStatus::AVAILABLE->value);
        self::assertSame('IN_TRANSIT', HuStatus::IN_TRANSIT->value);
        self::assertSame('BLOCKED', HuStatus::BLOCKED->value);
    }
}
